apps/console/src/features/identity-providers/components/forms/authenticators/saml-authenticator-settings-form.tsx

```diff
@@ -0,0 +1,814 @@
+                key: "IdPEntityId"
+            }),
+            IncludeProtocolBinding: findPropVal<boolean>({
+                defaultValue: castToBool(findMeta({ key: "IncludeProtocolBinding" })?.defaultValue) ?? false,
+                key: "IncludeProtocolBinding"
+            }),
+            IsAuthnRespSigned: findPropVal<boolean>({
+                defaultValue: castToBool(findMeta({ key: "IsAuthnRespSigned" })?.defaultValue) ?? false,
+                key: "IsAuthnRespSigned"
+            }),
+            IsLogoutEnabled: findPropVal<boolean>({
+                defaultValue: castToBool(findMeta({ key: "IsLogoutEnabled" })?.defaultValue) ?? false,
+                key: "IsLogoutEnabled"
+            }),
+            IsLogoutReqSigned: findPropVal<boolean>({
+                defaultValue: castToBool(findMeta({ key: "IsLogoutReqSigned" })?.defaultValue) ?? false,
+                key: "IsLogoutReqSigned"
+            }),
+            IsSLORequestAccepted: findPropVal<boolean>({
+                defaultValue: castToBool(findMeta({ key: "IsSLORequestAccepted" })?.defaultValue) ?? true,
+                key: "IsSLORequestAccepted"
+            }),
+            IsUserIdInClaims: findPropVal<boolean>({
+                defaultValue: castToBool(findMeta({ key: "IsUserIdInClaims" })?.defaultValue) ?? true,
+                key: "IsUserIdInClaims"
+            }),
+            LogoutReqUrl: findPropVal<string>({
+                defaultValue: findMeta({ key: "LogoutReqUrl" })?.defaultValue ?? "",
+                key: "LogoutReqUrl"
+            }),
+            NameIDType: findPropVal<string>({
+                defaultValue: findMeta({ key: "NameIDType" })?.defaultValue ?? "",
+                key: "NameIDType"
+            }),
+            RequestMethod: findPropVal<string>({
+                defaultValue: findMeta({ key: "RequestMethod" })?.defaultValue ?? "",
+                key: "RequestMethod"
+            }),
+            SPEntityId: findPropVal<string>({
+                defaultValue: findMeta({ key: "SPEntityId" })?.defaultValue ?? "",
+                key: "SPEntityId"
+            }),
+            SSOUrl: findPropVal<string>({
+                defaultValue: findMeta({ key: "SSOUrl" })?.defaultValue ?? "",
+                key: "SSOUrl"
+            }),
+            SignatureAlgorithm: findPropVal<string>({
+                defaultValue: findMeta({ key: "SignatureAlgorithm" })?.defaultValue ?? "RSA with SHA1",
+                key: "SignatureAlgorithm"
+            }),
+            commonAuthQueryParams: findPropVal<string>({
+                defaultValue: findMeta({ key: "commonAuthQueryParams" })?.defaultValue ?? "",
+                key: "commonAuthQueryParams"
+            })
+        } as SamlProperties;
+
+    }, []);
+
+    /**
+     * Form submission.
+     * @param values
+     * @param _ form
+     * @param __ callback
+     */
+    const onFormSubmit = (
+        values: { [ key: string ]: any },
+        _: FormApi,
```

Comment:
Lets drop the un-used args.

Reply:
Addressed this improvement with commit 1c606d3